lib/content-services/src/lib/dialogs/folder.dialog.scss

```diff
@@ -32,3 +32,14 @@
 .adf-dialog-action-button:disabled > span {
     color: var(--theme-text-color);
 }
+@media screen and (max-width: 380px){
+    .mat-form-field{
```

Comment:
This class will be applied to all the mat form field like that.  Same for the class below

Reply:
hi @eromano I have resolved the class issues, please have a look once.